src/i18n/en/translation.json

```diff
@@ -2244,7 +2244,7 @@
       "syncing_data": "Syncing data...",
       "syncing_data_description": "Unit data hasn't been synced yet. You can do it now by clicking the three dots menu and then '@:controller.units.sync_unit_info '.",
       "cannot_open_unit": "Cannot open unit",
-      "cannot_open_unit_description": "The unit {name} must be updated before opening it. Access the unit UI directly and go to System > Updates page.",
+      "cannot_open_unit_description": "The unit '{name}' needs to be updated before it can be opened. To update, open the three-dot menu in the unit list and select '@:controller.units.check_packages_updates ' and/or '@:standalone.update.update_system '. Alternatively, you can access the unit UI and navigate to '@:standalone.system.title ' > '@:standalone.update.title '.",
```

Comment:
```suggestion
      "cannot_open_unit_description": "The unit '{name}' needs to be updated before it can be opened. To update, open the three-dot menu in the unit list and select '@:controller.units.check_packages_updates ' and/or '@:standalone.update.update_system '. Alternatively, you can access the unit UI and navigate to '@:standalone.system.title > @:standalone.update.title '.",
```